packages/koa-bolt/src/index.spec.ts

```diff
@@ -0,0 +1,95 @@
+import Router from '@koa/router';
+import { events, ServerlessTester } from '@slack-wrench/fixtures';
+import { App, ExpressReceiver } from '@slack/bolt';
+import Koa, { Middleware } from 'koa';
+
+import koaBolt from './index';
+
+const identityRoute = (path: string): Middleware<any, any> => {
+  const route = new Router();
+
+  route.get(path, ctx => {
+    ctx.body = path;
+  });
+
+  return route.middleware();
+};
+
+describe('Koa Bolt Middleware', () => {
+  let app: Koa;
+  let bolt: App;
+  let receiver: ExpressReceiver;
```

Comment:
any reason to have these at this scope, instead of just `const`s in `setupFixtures`? AFAICT they're only used in `setupFixtures` - the only things that need to be exposed at this scope are the `handler` and `listener` I think?

Reply:
Nope, leftover from experiments. Good catch.